plugins/ruby-foreman-ansible/debian/gem.list

```diff
@@ -1,3 +1,3 @@
 # list of gem urls to download via Jenkins, space separated
-GEMS="https://rubygems.org/downloads/foreman_ansible-2.0.1.gem"
+GEMS="https://rubygems.org/downloads/foreman_ansible-2.2.4.gem"
 GEMS="$GEMS https://rubygems.org/downloads/foreman_ansible_core-2.0.2.gem"
```

Comment:
this should get updated to 2.1.1 now, I guess?

Reply:
I don't think that is necessary, the gemspec says ~> 2.0 but probably a good idea to update anyway

Reply:
in fact 2.0.2 does not require net-ssh so how about we keep it as is so we can release 2.2.5 and bump core dependency with next version where we'll have net-ssh unpinned?